### DISCUSSION TITLE
Custom footer in Calendar.

### DISCUSSION THREAD

**Discussion by eduardrassolov** [wojtekmaj/react-calendar#908] [Q&A]
Hi, 
Can I render some custom block in Calendar, looks like this footer?

<img width="600" alt="Знімок екрана 2023-11-08 о 22 06 49" src="https://github.com/wojtekmaj/react-calendar/assets/115376619/37ea2132-04bf-4609-99bf-a9e88089d3c4">

**Comment by ashcorah** [comment]
Did you find an answer to this?
